src/gitlab.js

```diff
@@ -23,25 +29,47 @@ export default class VerdaccioGitLab {
       }
     }
   ) {
-    this.users = {};
+    this.logger = stuff.logger;
 
+    this.users = {};
     this.config = config;
     this.stuff = stuff;
 
-    this.logger = stuff.logger;
+    if ((this.config.authCache || {}).enabled === false) {
```

Comment:
IMHO this should be disabled by default. It might confuse users @bufferoverflow whether do have old configuration.

Reply:
In many cases the will expect that the cache works in multiple processes and will drive to ticket asking why it does not work.

Reply:
Well, @bufferoverflow can take the final decision, but I think this plugin is not really usable out of testing without a cache, the amount of login calls performed on the upstream gitlab would be huge for any reasonable sized load.

This is untested, but I don't think there will be issues with multiple processes, just that more logins will happen (basically n times more, where n is the number of processes), but each process will keep their own copy of the cache.

I'd be more worried at the moment about the memory usage of each process with a bigger load :-)

Reply:
major version bump can be done after this is in, so using cache by default is fine